src/components/NavGrid.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+    <section class="nav__grid">
```

Comment:
Is there a reason this doesn't use the [Vuetify grid system](https://vuetifyjs.com/en/components/grids#grid-system)?